bioptim/examples/getting_started/models/soft_contact_sphere.bioMod

```diff
@@ -0,0 +1,57 @@
+version 4
+
+// DEFINITION DU SEGMENTS
+
+segment ground
+endsegment
+
+segment point
+     translations yz
+     rotations x
+    ranges
+    -10 10
+    -10 10
+    -pi pi
+    mass 20
+    inertia
+        1 0 0
+        0 1 0
+        0 0 1
+    com 0 0 0
+    mesh 0 0 0
+    mesh 0 0 0.08
+endsegment
+
+
+
+// Soft contacts
+
+  softcontact Contact1
+        parent point
+        type sphere
+        position 0 0 0
+        radius 0.08
+        stiffness 1e9
+        damping 2
+        muStatic 1
+        muDynamic 1
+        muViscous 0
+    endsoftcontact
+
+// Markers
+
+
+marker marker_point
+	    parent point
+	    position 0 0 0
+endmarker
+
+marker start
+    parent ground
+	position 0 0 9.33176954e-02
+endmarker
+
+marker end
+    parent ground
+    position 0 0.05 9.33176954e-02
```

Comment:
Are these values need to be that precise, why not just 0.09 or 0.093?

Reply:
it comes from the equilibrium of the sphere with the gound

Reply:
My worries is, if you need something that precise, is not this impossible to get in practice?